packages/server/customer-os-api/graph/resolver/organization.resolvers.go

```diff
@@ -190,37 +190,37 @@ func (r *mutationResolver) OrganizationUpdate(ctx context.Context, input model.O
 	}
 	fieldsMask := []organizationpb.OrganizationMaskField{}
 	if input.Patch != nil && *input.Patch {
-		if input.Name != "" {
+		if utils.IfNotNilString(input.Name) != "" {
```

Comment:
shouldn't be
if input.Name != nil 
?

maybe you want to remove the name for an org and become "unknown "

Reply:
will do it after first round of release.